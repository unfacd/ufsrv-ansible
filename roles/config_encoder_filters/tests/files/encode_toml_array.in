arr1:
  - some text
  -
    - 123
    - 987.654
  - yes
  - "false"
