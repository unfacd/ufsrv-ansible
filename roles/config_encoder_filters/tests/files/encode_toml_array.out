arr1 = ["some text", [123, 987.654], true, "false"]
